.circleci/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM circleci/clojure:lein-2.7.1
+LABEL maintainer="hlship@gmail.com"
+
+# Based on notes here:
+# https://www.metachris.com/2017/01/how-to-install-nodejs-7-on-ubuntu-and-centos/
+
+RUN curl -sL https://deb.nodesource.com/setup_7.x | sudo bash -
+RUN sudo apt-get install -y nodejs
+
+# docker build -t hlship/circleci-lein-node:2.7.1 .
```

Comment:
Also, complications from following the Apollo model:  initial handshake, keep alive.  

Also, need to track things so that we can cleanly shutdown subscriptions when client closes.

It's a tricky thing.  I think we're (on the path to) doing it right.